fern/openapi-ir/definition/ir.yml

```diff
@@ -0,0 +1,136 @@
+types: 
+  IntermediateRepresentation: 
+    properties: 
+      displayName: optional<string>
+      environments: list<Environment>
+      endpoints: list<Endpoint>
+      schemas: map<SchemaId, Schema>
+      schemasToTags:
+        type: map<SchemaId, list<TagId>>
+        docs: Which schemas are referenced by which tags
+
+  WithDescription: 
+    properties: 
+      description: optional<string>
+
+  Environment: 
+    extends: WithDescription
+    properties: 
+      name: optional<string>
+      url: string
+
+  TagId: 
+    type: string
+    docs: The name of the tag.
+  
+  TagInfo: 
+    properties: 
+      description: string
+      displayName: optional<string>
+
+  SchemaId: 
+    type: string
+    docs: The name of the schema.
```

Comment:
name or id? are these the same thing in openapi? would just pick one for clarity